nc-news-frontend/src/pages/Article.jsx

```diff
@@ -24,13 +26,40 @@ const Article = () => {
             fetchArticleAndComments();
       }, [article_id]);
 
+      const handleSubmit = (e) => {
+        e.preventDefault();
     
+        const requestedBody = {
+          username,
+          body,
+        };
+        setLoading(true)
```

Comment:
I would not have this setLoading - it make the screen very flashy when posting. Technically you've not reloading the comments. 

if you wanted a loading, id have one just for the comment box, e.g. theres a loading message instead of it saying submit, or something like that. I think it would make the whole UX a bit better without that flashing of the loading then comments you currently have after posting. 
Still keep this in place for when you fetch the comments though when the page first loads.